org.kde.kcolorchooser.json

```diff
@@ -0,0 +1,33 @@
+{
+    "id": "org.kde.kcolorchooser",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kcolorchooser",
+    "finish-args": [
+        "--share=ipc",
```

Comment:
if you run it on the command line, it will tell you:

For example:
```
libEGL warning: wayland-egl: could not open /dev/dri/renderD128 (No such file or directory)
```

(could be a different message depending on Wayland or the GPU drivers)

Reply:
You can try running it on wayland. I've just did that and I get it indeed:
```
libEGL warning: wayland-egl: could not open /dev/dri/renderD128 (No such file or directory)
```
Thus this is needed.